ironfish/src/consensus/verifier.test.ts

```diff
@@ -85,7 +85,8 @@ describe('Verifier', () => {
       })
     })
 
-    it('rejects a block with miners fee as second transaction', async () => {
+    //this actually fails for transaction fee in the miners slot
+    it('rejects a block with transaction fee as first transaction', async () => {
```

Comment:
maybe "non-miner's transaction fee"? "transaction fee" can be a bit ambiguous between miner's fees and standard transaction fees.